src/Rule.php

```diff
@@ -76,15 +76,15 @@ public function translationLocale(string $locale): self
         return $new;
     }
 
-    public function translateMessage(string $message, array $arguments = []): string
+    protected function translateMessage(string $message, array $params = []): string
```

Comment:
I think it should be arguments in this case. See https://trello.com/c/R6DBDMdj/59-make-it-consistent-about-naming-parameters-vs-arguments

Reply:
It isn't just parameters or arguments of function, it parameters of message.
Also the second argument in `Translator::translate` named as `parameters` https://github.com/yiisoft/i18n-translator/blob/master/src/Translator.php#L59